cdap-ui/app/cdap/components/Tags/Tag/index.js

```diff
@@ -79,5 +80,9 @@ export default class Tag extends Component {
 Tag.propTypes = {
   value: PropTypes.string,
   scope: PropTypes.string,
-  onDelete: PropTypes.func
+  onDelete: PropTypes.func,
```

Comment:
nit: Since we are changing the `propTypes` can we make this and the defaultProps as `static` properties and remove constructor and have `state` as class property? It should be a no-risk change